backend/app/tasks.py

```
@@ -2155,13 +2161,15 @@ def process_single_app(item, target, instance_item, update_time, result=None):
             result.add_update()
 
         logger.debug(f"Retrieving detailed data for application: {app_id}")
+        data_retrieved = True
         try:
             data = item.get_data()
             logger.debug(f"Successfully retrieved detailed data for application: {app_id}")
         except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)